src/spineml/simulator/logOutput.cc

```diff
@@ -274,16 +233,73 @@ void SpineMLSimulator::LogOutput::AnalogueNetwork::record(double, unsigned int t
     }
 }
 
+//----------------------------------------------------------------------------
+// SpineMLSimulator::LogOutput::AnalogueNetwork
+//----------------------------------------------------------------------------
+SpineMLSimulator::LogOutput::AnalogueNetwork::AnalogueNetwork(const pugi::xml_node &node, double dt,
+                                                              const std::string &port, unsigned int popSize,
+                                                              const filesystem::path &logPath,
+                                                              const ModelProperty::Base *modelProperty)
+    : AnalogueExternal(node, dt, port, popSize, logPath, modelProperty)
+{
+    // Check size determined by indices/population size matches attribute
+    const unsigned int size = getIndices().empty() ? popSize : (unsigned int)getIndices().size();
```

Comment:
Probably better to use a ``size_t`` here. MSVC and gcc use different types for ``size_t``

Reply:
...or ``auto``

Reply:
Yeah - population sizes **should** definitely be ``size_t``. The problem is that a lot of GeNN and hence SpineML  uses ``unsigned int`` for population size and I'm not sure if it's worth fixing this particular bit in this P.R.

Reply:
Ok, fair enough. It's probably not a bad thing to force population sizes to be ``unsigned int`` on Windows and *nix, so that they're 32 bits on all platforms.